CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ set(MATERIALX_MAJOR_VERSION 1)
 set(MATERIALX_MINOR_VERSION 39)
 set(MATERIALX_BUILD_VERSION 2)
 set(MATERIALX_LIBRARY_VERSION ${MATERIALX_MAJOR_VERSION}.${MATERIALX_MINOR_VERSION}.${MATERIALX_BUILD_VERSION})
+math(EXPR MATERIALX_BUILD_IDENTIFIER "${MATERIALX_MAJOR_VERSION} * 10000 + ${MATERIALX_MINOR_VERSION} * 100 + ${MATERIALX_BUILD_VERSION}")
```

Comment:
Since there are many different strategies for combining three version numbers into a monotonically-increasing "version index", we may want to keep this new logic into the corresponding `AppleFrameworkBuild.zsh.in` file for now, so that we're not officially stating that this is the approach that MaterialX will apply universally in the future.

Here's some additional discussion of different options for this combining logic, including the approaches used in Vulkan and USD, with the most recent posts leaning towards the Vulkan approach:

https://github.com/AcademySoftwareFoundation/MaterialX/issues/1609
`

Reply:
I need it in the CMake, since the CMake is what generates the final `plist` which is where its used.
I'll move it into the block that generates the plist though so its not a global like it is now.